src/l1-resources/cluster/index.ts

```diff
@@ -158,6 +165,7 @@ export function toJson_CfnClusterProps(
     ConnectionStrings: toJson_ConnectionStrings(obj.connectionStrings),
     DiskSizeGB: obj.diskSizeGb,
     EncryptionAtRestProvider: obj.encryptionAtRestProvider,
+    GlobalClusterSelfManagedSharding: obj.globalClusterSelfManagedSharding,
```

Comment:
Is it worth updating an example?

Reply:
As this was an advanced feature. We chose to not update the example.